### ISSUE TITLE
Outlook 2016 looses connection to Exchange server

### ISSUE THREAD

**Issue by fgcgithub** [stascorp/rdpwrap#185]
I was trying rdpwrap on Windows 10 build 14393.447 64-bit, works great - but I ran into an Outlook problem. Outlook couldn't establish a connection to its Exchange server.

**Comment by jsnip4** [comment]
I have noticed the same issue but I also notice that the network icon by the clock shows an orange mark that there is no connectivity. I have the disable my network adaptor and then enable it. Then it comes back fine and outlook works again.

**Comment by binarymaster** [comment]
@fgcgithub @jsnip4 do you use any VPN (PPPoE, PPTP) connections?

**Comment by jsnip4** [comment]
@fgcgithub @binarymaster No I do not use any VPN. I tested this with 1.6 and 1.6.1 and I have the issue but not every time, only sometimes when installing or uninstalling RDP Wrap. I get the little orange triangle with the exclamation point telling me "No Internet access". Even though it shows this, I can get online just fine but MS Outlook shows "Disconnected" and will not connect until that triangle goes away and for me I have to disable the network adapter and then enable it again.

**Comment by binarymaster** [comment]
This may be caused by the installer, since it terminates shared system services along with Terminal Server.

**Comment by jsnip4** [comment]
binarymaster, do you have a solution to fix this? Thanks

**Comment by binarymaster** [comment]
@jsnip4 I have no idea for now. Maybe installer could not terminate shared services, but properly stop them and later restart again.

**Comment by jsnip4** [comment]
Can you tell me which services it is stopping and trying to start? I have a remote control software similar to TeamViewer in which I install RDP Wrap. If I could find out which services I could have my software make sure those services are running after installation or update of RDP Wrap.

**Comment by binarymaster** [comment]
Use command `tasklist /svc` and look for `TermService` process.

**Comment by jsnip4** [comment]
The service named NlaSvc is causing the problem. Your installer and uninstaller stops and starts this service. I have found that if I open a command prompt with administrator rights and run: net stop NlaSvc /y, (it asks to stop another service "netprofm" so that is why I added the /y) and then the NlaSvc  service automatically restarts itself and the "No Internet access" message goes away from the Network icon and everything works fine including Outlook.
